tests/dummy/app/query-params/route.js

```diff
@@ -0,0 +1,21 @@
+import Ember from 'ember';
+import route from 'ember-redux/route';
+
+var queryParams = {
+  foo: {
+    refreshModel: false
+  }
```

Comment:
why set refreshModel to false here? trying to read thru everything to better understand this so forgive me if I've totally missed it